mlflow/models/evaluation/base.py

```diff
@@ -926,6 +926,12 @@ def evaluate(
         - **log_metrics_with_dataset_info**: A boolean value specifying whether or not to include
           information about the evaluation dataset in the name of each metric logged to MLflow
           Tracking during evaluation, default value is True.
+        - **pos_label**: The positive label used to compute binary classification metrics such as
+          precision, recall, f1, etc (default: ``None``). This parameter is only used for binary
```

Comment:
Should `pos_label` default to 1 for binary classification models?

Reply:
`eval_and_log_metrics` uses `None` by default regardelss of the model type.

Reply:
In `eval_and_log_metrics`, it will set `average` to `weighted` by default, in which case the `pos_label` doesn't matter. I think that's wrong, and we do not need to be backward compatible to that. Setting `pos_label` default to 1 makes sense.

Reply:
That makes sense, I will update the code.